cpp/test/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ cxx_add_compile_options(MSVC /wd4310)
 # Core unit tests don't build under VS2013 without this.
 cxx_add_compile_options(MSVC /EHa)
 
-if (NOT BOND_SKIP_CORE_TESTS)
+if (NOT BOND_SKIP_COMPAT_TESTS)
```

Comment:
Done

---
In reply to: [130746141](https://github.com/Microsoft/bond/pull/540#discussion_r130746141) [](ancestors = 130746141)